.evergreen/.evg.yml

```diff
@@ -1069,8 +1069,8 @@ buildvariants:
     - name: "plain-auth-test"
 
 - matrix_name: "aws-auth-test"
-  matrix_spec: { jdk: "jdk11" }
-  display_name: "MONGODB-AWS Auth test"
+  matrix_spec: { auth: "noauth", ssl: "nossl", jdk: "jdk11", version: ["4.4", "latest"], os: "ubuntu" }
+  display_name: "MONGODB-AWS Auth test ${version} ${os}"
```

Comment:
Change `os` to `jdk`.  We typically don't display OS name since we only test on Linux anyway.

Reply:
Done